pkg/oc/bootstrap/docker/down.go

```diff
@@ -75,7 +75,7 @@ func (c *ClientStopConfig) Stop(out io.Writer) error {
 		return err
 	}
 	for _, name := range names {
-		if _, _, err = dockertools.ParseDockerName(name); err != nil {
+		if _, err = parseDockerName(name); err != nil {
```

Comment:
👍 guess this is reducing the external dependencies

Reply:
> guess this is reducing the external dependencies

well, in this case it was a kube function that was removed.

Reply:
ouch. Make it consumable by downstream (us) again?